elyra/pipeline/kfp/processor_kfp.py

```diff
@@ -1062,7 +1062,7 @@ def _compose_container_command_args(
         # meant for the opendatahub-io/Data-science-pipelines v2 use case
         # and should not be opened as PR against upstream Elyra
         elyra_github_org = os.getenv("ELYRA_GITHUB_ORG", "opendatahub-io")
-        elyra_github_branch = os.getenv("ELYRA_GITHUB_BRANCH", "jupyterlab4" if "dev" in __version__ else __version__)
+        elyra_github_branch = os.getenv("ELYRA_GITHUB_BRANCH", "main" if "dev" in __version__ else f"v{__version__}")
```

Comment:
```suggestion
        elyra_github_branch = os.getenv("ELYRA_GITHUB_BRANCH", "v4.0.x" if "dev" in __version__ else f"v{__version__}")
```

Should we point it to the branch, so that in future,
if main based files go ahead, at least this would safe guard the users ?